### ISSUE TITLE
KeyError in varLib.cff.merge_PrivateDicts

### ISSUE THREAD

**Issue by miguelsousa** [fonttools/fonttools#1632]
Getting the error below while trying to build the fonts at https://github.com/adobe-fonts/adobe-blank-vf

Test case attached. The command was,

    fonttools varLib --master-finder fonts/{stem}.otf AdobeBlankVF.designspace

```
Traceback (most recent call last):
  File "fonttools", line 10, in <module>
    sys.exit(main())
  File "fontTools/__main__.py", line 29, in main
    runpy.run_module(mod, run_name='__main__')
  File "runpy.py", line 208, in run_module
    return _run_code(code, {}, init_globals, run_name, mod_spec)
  File "runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "fontTools/varLib/__main__.py", line 7, in <module>
    sys.exit(main())
  File "fontTools/varLib/__init__.py", line 1064, in main
    optimize=options.optimize
  File "fontTools/varLib/__init__.py", line 924, in build
    _add_CFF2(vf, model, master_fonts)
  File "fontTools/varLib/__init__.py", line 767, in _add_CFF2
    merge_region_fonts(varFont, model, ordered_fonts_list, glyphOrder)
  File "fontTools/varLib/cff.py", line 310, in merge_region_fonts
    merge_PrivateDicts(top_dicts, cvData.vsindex_dict, model, fd_map)
  File "fontTools/varLib/cff.py", line 183, in merge_PrivateDicts
    sub_model, model_keys = vsindex_dict[vsindex]
KeyError: 0
```
[AdobeBlankVF.zip](https://github.com/fonttools/fonttools/files/3245532/AdobeBlankVF.zip)

**Comment by kenlunde** [comment]
I am sure that it has something to do with its _blank_ glyphs, as the corresponding **black** VFs build successfully.

**Comment by readroberts** [comment]
I can reproduce this. Am looking at it now.

**Comment by readroberts** [comment]
varLib.cff.py didn't support building a variation font where none of the glyphs have any variation. I have a fix for this, will submit after more testing.

**Comment by anthrotype** [comment]
Fixed by #1642
